db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_03_18_181006) do
+ActiveRecord::Schema.define(version: 2019_03_21_180503) do
```

Comment:
Oh oh I think schema has been accidentally committed here

Reply:
Ooops, I've uncommited (is that a word?) that file. Specs are passing again now too